demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -87,10 +94,13 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
             self.is_valid_description(beta_integration=False),
         ]
 
+        answers3 = []
+        if support_level in LEVEL_SUPPORT_OPTIONS_3:
+            answers3 = [self.is_valid_max_fetch_and_first_fetch()]
```

Comment:
why not append to `answers`? - your life might be made easier by y previous comment

Reply:
wanted to make it ready for later changes but I thing I will append it for now